torch/distributed/rpc/server_process_global_profiiler.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/python3
+
+import torch
+
+from . import (
+    _disable_server_process_global_profiler,
+    _enable_server_process_global_profiler,
+)
+
+
+def enable_server_process_global_profiler(config):
+    _enable_server_process_global_profiler(config)
+
+
+def disable_server_process_global_profiler():
+    profile_ranges = _disable_server_process_global_profiler()
+
+    process_global_function_events = []
+    for profile_range in profile_ranges:
+        thread_local_function_events = torch.autograd.profiler.parse_cpu_trace(
+            profile_range
+        )
+        process_global_function_events.extend(thread_local_function_events)
+
+    process_global_function_events.sort(
+        key=lambda function_event: [
+            function_event.cpu_interval.start,
+            -(function_event.cpu_interval.end),
+        ]
+    )
+
+    return torch.autograd.profiler.EventList(
```

Comment:
Is it intentional to make `disable_server_process_global_profiler` the only way to retrieve the event list? What if I'd like to retrieve the list but don't yet want to disable it?

Reply:
Yes, it should be the only way. It's the counterpart of autograd profiler, disableProfiler, API. I don't think there is other use cases.